code/modules/urist/modules/jungle/planet_turfs.dm

```diff
@@ -1,32 +1,32 @@
 /turf/simulated/floor/planet
-	var/animal_spawn_chance = 0
-	var/plants_spawn_chance = 0
-	var/small_trees_chance = 0
-	var/large_trees_chance = 0
-	var/misc_plant_spawn_chance = 0
-	var/trap_spawn_chance = 0
+	var/animal_spawn_chance = 0 //% chance to spawn animals. actual percentage is roughly divided in half bc i was an idiot 9 years ago
+	var/plants_spawn_chance = 0 //%chance to spawn plants from the general plant icon database. this includes fruit bearing plants
+	var/small_trees_chance = 0 //% chance to spawn small trees
+	var/large_trees_chance = 0 //% chance to spawn large trees
+	var/misc_plant_spawn_chance = 0 //misc plant spawn chance - this is a specifically defined type
+	var/trap_spawn_chance = 0 //trap spawns??? idfk. check below for the ancient trap code
```

Comment:
This is not directed at you @CaraS212, just a general note - it would likely be a much better idea to switch from doing stuff like this, per-tile gen, to a more decoupled, event-driven-ish approach where a tile's Initialize() pings a (datum-based, most likely) map generator that it needs processing.

This would let mapgen act a bit smarter, because it could have access to higher-level information about the map such as neighboring tiles, which would make it easier to generate e.g. random traversable paths/clearings instead of hardcoding them.

(...also I want to port WFC for this once I get it working in DM 👀)